code/modules/client/verbs/ooc.dm

```diff
@@ -112,6 +112,10 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 			discordsendmsg("ooc", "[discord_ooc_tag](OOC) **[sayer]:** [msg]")
 		if(CHAT_TYPE_DEADCHAT) // don't send these until a round is finished
 			if(SSticker.current_state == GAME_STATE_FINISHED)
+				var/regex/R = regex("<span class=' '>(.+)</span>\"")
```

Comment:
`.` will not work for this, it needs to be [\s\S] to match any character including whitespace